src/auro-popover.js

```diff
@@ -9,16 +9,20 @@ import { LitElement, html, css } from "lit-element";
 import "focus-visible/dist/focus-visible.min.js";
 import styleCss from "./style-css.js";
 
+import {createPopper} from '@popperjs/core';
+
 // build the component class
+const POPOVER_OFFSET_MIN = 0;
+const POPOVER_OFFSET_MAX = 18;
+
 class AuroPopover extends LitElement {
-  // constructor() {
-  //   super();
-  // }
 
-  // function to define props used within the scope of thie component
+  // function to define props used within the scope of this component
   static get properties() {
     return {
-      cssClass:   { type: String }
+      placement:  { type: String },
+      for:        { type: String },
+      tooltip:    { type: Object }
```

Comment:
Why is this part of the public API. I don't see why/how you would use this at the WC level?